### PR TITLE
Add aliases for most common objects

### PR DESCRIPTION
**Pull request by PurityLake**
Refers to #3353

Added aliases to `pygame` module:
* Joystick
* Font
* Event
* Channel
* Clock

### CODE REVIEW

**Review by Starbuck5 — COMMENTED** [review]
It can't be implemented *quite* this simply, because pygame is built to be modular, and to be okay with certain modules not being compiled with it.

Rect is required, but font is not.

These should go in the try excepts for their respective modules.

**Review by Starbuck5 — COMMENTED** [review]
Some of these will take a while to get used to... But it'll be a cool change!

I've just realized that these should probably get tiny unit tests, to prevent this new from behavior from unintentional regressions.

It could be as simple as asserting pygame.joystick.Joystick == pygame.Joystick, but for each one.

Oh shoot, how could I forget! This also will need documentation and type hinting support.

**Comment by PurityLake** [comment]
I'll hop to that then

**Comment by Starbuck5** [comment]
This still needs tests and type hints.

**Comment by PurityLake** [comment]
I believe that should be it for this PR unless I have overlooked something.

File: buildconfig/stubs/pygame/__init__.pyi [hunk 1]
```diff
@@ -52,6 +52,10 @@ from .math import Vector2 as Vector2, Vector3 as Vector3
 from .cursors import Cursor as Cursor
 from .bufferproxy import BufferProxy as BufferProxy
 from .mask import Mask as Mask
+from .event import Event as Event
+from .font import Font as Font
+from .mixer import Channel as Channel
+from .time import Clock as Clock
```
Comment (MyreMylar) [inline buildconfig/stubs/pygame/__init__.pyi:58]: `Joystick` is missing here.

**Review by MyreMylar — CHANGES_REQUESTED** [review]
Seems fine, except `Joystick` is missing from the typestubs.

File: buildconfig/stubs/pygame/__init__.pyi (hunk 1 above)
Comment (PurityLake) [reply buildconfig/stubs/pygame/__init__.pyi:58]: That should be fixed now

**Review by MyreMylar — APPROVED** [review]
LGTM 👍

**Review by Starbuck5 — COMMENTED** [review]
The `versionchanged` labels should point to 2.1.4 now

**Comment by PurityLake** [comment]
@Starbuck5 I have made that change to the docs

File: buildconfig/stubs/pygame/__init__.pyi [hunk 2]
```diff
@@ -51,6 +51,11 @@ from .math import Vector2 as Vector2, Vector3 as Vector3
 from .cursors import Cursor as Cursor
 from .bufferproxy import BufferProxy as BufferProxy
 from .mask import Mask as Mask
+from .event import Event as Event
```
Comment (zoldalma999) [inline buildconfig/stubs/pygame/__init__.pyi:54]: As mentioned in the comment at the top of the file, this is an autogenerated file. Please edit `buildconfig/stubs/gen_stubs.py` instead

File: src_py/__init__.py [hunk 3]
```diff
@@ -252,6 +258,7 @@ def Overlay(format, size):  # pylint: disable=unused-argument
 
 try:
     import pygame.mixer
+    from pygame.mixer import Channel
```
Comment (zoldalma999) [inline src_py/__init__.py:261]: `Sound` is missing from here. It needs a test, stubs, and mention in the docs as well (unless it is intentional?).

File: src_py/__init__.py [hunk 4]
```diff
@@ -129,6 +130,7 @@ def warn(self):
 
 try:
     import pygame.event
+    from pygame.event import Event
```
Comment (zoldalma999) [inline src_py/__init__.py:133]: If the event module is not available, the Event class won't exist. Other modules where the import fails, they define a dummy function (see Cursor for example).

**Pull request closed without merging**